tutorial3_distributions/demo/main.js

```diff
@@ -1,133 +1,127 @@
 // CONSTANTS
-const width = window.innerWidth * 0.8;
-const height = window.innerHeight * 0.8;
-const margin = { top: 20, bottom: 40, left: 40, right: 40 };
-const radius = 3;
+const constants = {
+  width: window.innerWidth * 0.7,
+  height: window.innerHeight * 0.7,
+  margin: { top: 20, bottom: 40, left: 40, right: 40 },
+  radius: 3,
+}
 
 // APPLICATION STATE
-let data = [];
-let selectedState;
+let state = {
+  data: [],
+  selectedParty: "D"
+}
```

Comment:
this was as discussed but I took it a step further with `constants`... it just feels more compact (and helps identify variables). I would even make the claim that we could do it with `globals` too but will leave that up to you.